salesapp.js

```diff
@@ -2,6 +2,21 @@
 
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm'];
 var cookieStores = [];
+var addStoreForm = document.getElementById('add_store_form');
+var tableEl = document.getElementById('cookieTable');
+
+addStoreForm.addEventListener('submit', function(event) {
+  event.preventDefault();
+  var minCustPerHour = parseInt(document.getElementById('min_cust_per_hour').value),
+    maxCustPerHour = parseInt(document.getElementById('max_cust_per_hour').value),
+    avgCookiesPerCust = parseFloat(document.getElementById('avg_cookies_per_hour').value),
+    storeName = document.getElementById('shop_location').value;
```

Comment:
You can use `document.getElementById(<form element id>).value` to grab data from your form elements like you did above, or you can use `event.target.<form element name>.value`. Either way works, but the latter will only pull information from *this* form